libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -127,14 +127,11 @@ public boolean onBackPressed() {
    * @param outState to store state variables
    */
   public void onSaveInstanceState(Bundle outState) {
-    if (summaryBehavior != null) {
```

Comment:
Should we still account for this logic in the instance state object?  I believe this was added from a crash that @Guardiola31337 found.

Reply:
For reference https://github.com/mapbox/mapbox-navigation-android/blob/092f807e18cdc48ef6c908853c84209005df93c5/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java#L131-L133 included in https://github.com/mapbox/mapbox-navigation-android/pull/959

Reply:
@danesfeder I added the logic into the `NavigationViewInstanceState` object